related-projects.md

```diff
@@ -0,0 +1,63 @@
+<!--
+SPDX-FileCopyrightText: Copyright 2022 Arm Limited and/or its affiliates <open-source-office@arm.com>
+SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+
+# Known implementations and projects related to the PSA Certified APIs
+
+Updated Dec 2022.
+ 
+## Software implementations
+
+- The reference implementation for PSA Crypto API can be found inside Mbed TLS:
+  [https://mbed-tls.readthedocs.io/en/latest/](https://mbed-tls.readthedocs.io/en/latest/)
```

Comment:
On other pages on this website, the URL is shown without the "https://" prefix, and without any trailing "/":

```suggestion
  [mbed-tls.readthedocs.io/en/latest](https://mbed-tls.readthedocs.io/en/latest/)
```

This reduces the visual clutter and makes it easier to see the important part of the URL.

I think this page will look better if this is done here too.